standalone-metastore/metastore-common/src/main/thrift/hive_metastore.thrift

```diff
@@ -2651,6 +2651,7 @@ PartitionsResponse get_partitions_req(1:PartitionsRequest req)
 
   // Transaction and lock management calls
   // Get just list of open transactions
+  //Deprecated use get_open_txns_req
```

Comment:
nit: Could you please add 1 space here? :)

Reply:
Added